LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016
+Copyright (c) 2017
```

Comment:
ember-cli made a mistake here (and we are revertting the update to the `LICENSE.md` blueprint).  

@Panman8201 - Can you change this to `2016-2017` instead of just updating to 2017?